lambda_src/drivers/process_smtp.py

```diff
@@ -1,16 +1,72 @@
+import json
 import smtplib
 import ssl
 from email.mime.multipart import MIMEMultipart
 from email.mime.text import MIMEText
+from typing import Dict, Tuple, Any
 
 from ..vault import decrypt_if_encrypted
 
 
+def parse_smtp_creds(auth_dict: Dict) -> Tuple[Any, Any, Any]:
```

Comment:
could we be a little more prescriptive and also make sure we can't possibly confuse some other drivers' credentials to be re-used here? maybe e.g. add a `"driver": "smtp"` key or just force `smtp_host` to make sure another driver wouldn't re-use these credentials?

sooner or later we'll need to document this and having a single option would make things simpler there, as well. people shouldn't be guessing here, and docs should give them one right answer.

that said, do appreciate the cool next/map combo, of course :)